### PR TITLE
Usage of API 2 for job bans

### PR DESCRIPTION
**Pull request by Wirewraith**
## About the PR 

Change job ban system to use the new API system

### CODE REVIEW

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -704,7 +704,7 @@ var/global/noir = 0
 			if (src.level >= LEVEL_SA)
 				var/M = href_list["target"]
 				var/job = href_list["type"]
-				var/list/cache = apiHandler.queryAPI("jobbans/get/player", list("ckey"=M), 1)[M]
+				var/list/cache = jobban_get_for_player(M)
```
Comment (ZeWaka) [inline code/modules/admin/admin.dm:707]: usage does not match definition, i assume M is a mob not a ckey

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/modules/admin/admin.dm (hunk 1 above)
Comment (Wirewraith) [reply code/modules/admin/admin.dm:707]: It's not, it's a ckey apparently

Comment (ZeWaka) [reply code/modules/admin/admin.dm:707]: please change it from M then

Comment (Wirewraith) [reply code/modules/admin/admin.dm:707]: No, out of scope

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**